### ISSUE TITLE
Error adding libraries to a project with an Aggregate target

### ISSUE THREAD

**Issue by sgriner** [keithpitt/vendor#5]
When running `vendor install` on a project with an aggregate target, I get the following error:

/Library/Ruby/Gems/1.8/gems/vendor-0.1.4/lib/vendor/xcode/project.rb:50:in `const_get': uninitialized constant Vendor::XCode::Proxy::PBXAggregateTarget (NameError)
    from /Library/Ruby/Gems/1.8/gems/vendor-0.1.4/lib/vendor/xcode/project.rb:50:in`reload'
    from /Library/Ruby/Gems/1.8/gems/vendor-0.1.4/lib/vendor/xcode/project.rb:39:in `map'
    from /Library/Ruby/Gems/1.8/gems/vendor-0.1.4/lib/vendor/xcode/project.rb:49:in`each'
    from /Library/Ruby/Gems/1.8/gems/vendor-0.1.4/lib/vendor/xcode/project.rb:49:in `map'
    from /Library/Ruby/Gems/1.8/gems/vendor-0.1.4/lib/vendor/xcode/project.rb:49:in`reload'
    from /Library/Ruby/Gems/1.8/gems/vendor-0.1.4/lib/vendor/xcode/project.rb:29:in `initialize'
    from /Library/Ruby/Gems/1.8/gems/vendor-0.1.4/lib/vendor/cli/app.rb:89:in`new'
    from /Library/Ruby/Gems/1.8/gems/vendor-0.1.4/lib/vendor/cli/app.rb:89:in `install'
    from /Library/Ruby/Gems/1.8/gems/thor-0.14.6/lib/thor/task.rb:22:in`send'
    from /Library/Ruby/Gems/1.8/gems/thor-0.14.6/lib/thor/task.rb:22:in `run'
    from /Library/Ruby/Gems/1.8/gems/thor-0.14.6/lib/thor/invocation.rb:118:in`invoke_task'
    from /Library/Ruby/Gems/1.8/gems/thor-0.14.6/lib/thor.rb:263:in `dispatch'
    from /Library/Ruby/Gems/1.8/gems/thor-0.14.6/lib/thor/base.rb:389:in`start'
    from /Library/Ruby/Gems/1.8/gems/vendor-0.1.4/bin/vendor:14
    from /usr/bin/vendor:19:in `load'
    from /usr/bin/vendor:19

**Comment by keithpitt** [comment]
I think I have a fix in master for this. Could you try?

``` bash
cd ~/repos
git clone git://github.com/keithpitt/vendor.git
sudo ln -s /Users/you/repos/vendor/bin/vendor /usr/bin/vendor_dev
```

Then `vendor_dev install` to use the development copy.

**Comment by sgriner** [comment]
It's closer.
Now I get:

```
Installing JSONKit 1.4
Could not find 'Vendor::XCode::Proxy::PBXSourcesBuildPhase' build phase for target 'GenGitVersion'
```

**Comment by keithpitt** [comment]
Hrm.. Could you upload your XCode project somewhere and send me the link privately? I think there is a file type in your project that I don't know about.

**Comment by keithpitt** [comment]
Fixed in 92c5701cf751ff87b981c245291d254df5050053

**Comment by sgriner** [comment]
Works great. Thanks!

**Comment by keithpitt** [comment]
I have found another small bug with this when adding a library that has a framework, I'm going to repoen until I fix that issue.

**Comment by keithpitt** [comment]
Fixed in 4a885c8
